types/stripe/index.d.ts

```diff
@@ -8,17 +8,10 @@
 
 /// <reference types="node" />
 
-declare module 'stripe' {
-    var out: StripeNode.StripeExport;
-    export = out;
-}
+declare function StripeNode(apiKey: string, version?: string): StripeNode.Stripe;
+export = StripeNode;
```

Comment:
Don't know if it matters, but they call this `stripePackage` in their readme...

[ref](https://github.com/stripe/stripe-node#usage)

```js
import stripePackage from 'stripe';
const stripe = stripePackage('sk_test_...');
```

Reply:
Although, in the code it's named `Stripe` and it really seems like it's supposed to be a class, and not a function, even though their documentation kinda leads us on to think it's a function.

https://github.com/stripe/stripe-node/blob/a22a89c2e765d2548929d3d42dc0d7c4c4f6c19c/lib/stripe.js#L76-L109

Reply:
This also would only be reflected in TypeScript if we declare it a class I think?

```js
const Stripe = require('stripe')
const stripe = new Stripe('...')

stripe instanceof Stripe
//=> true
```

Reply:
The more I look the more I'm leaning against something like

```typescript
class Stripe {
  // ...
}

export = Stripe
```